### PR TITLE
Relationship test

### PR DESCRIPTION
**Pull request by WadeNaughton**
-relationship tests created
-validations test created
-features test that aren't needed added.

### CODE REVIEW

**Review by arnaldoaparicio — COMMENTED** [review]
Cool.

File: app/models/customer.rb [hunk 1]
```diff
@@ -1,3 +1,6 @@
 class Customer < ApplicationRecord
   has_many :invoices
+
+  validates :first_name, presence: true
```
Comment (tjroeder) [inline app/models/customer.rb:4]: Validations look good.

File: spec/features/items/index.html.erb_spec.rb [hunk 2]
```diff
@@ -18,11 +18,5 @@
     ])
   end
 
-  it "renders a list of items" do
```
Comment (tjroeder) [inline spec/features/items/index.html.erb_spec.rb:21]: Get that minitest outta here! Nice work.

**Review by tjroeder — APPROVED** [review]
Changes look good. I removed the merchants show page as it is no longer needed.

**Pull request merged**